eva_submission/eload_ingestion.py

```diff
@@ -108,16 +110,36 @@ def fill_vep_versions(self, vep_cache_assembly_name=None):
             if ('vep' in self.eload_cfg.query(self.config_section)
                     and assembly_accession in self.eload_cfg.query(self.config_section, 'vep')):
                 continue
-            vep_version, vep_cache_version, vep_species = get_vep_and_vep_cache_version(
+            vep_version, vep_cache_version = get_vep_and_vep_cache_version(
                 self.mongo_uri,
                 self.eload_cfg.query(self.config_section, 'database', assembly_accession, 'db_name'),
                 assembly_accession,
                 vep_cache_assembly_name
             )
+            vep_species = self.get_species_name_from_ncbi(assembly_accession)
             self.eload_cfg.set(self.config_section, 'vep', assembly_accession, 'version', value=vep_version)
             self.eload_cfg.set(self.config_section, 'vep', assembly_accession, 'cache_version', value=vep_cache_version)
             self.eload_cfg.set(self.config_section, 'vep', assembly_accession, 'species', value=vep_species)
 
+    def get_species_name_from_ncbi(self, assembly_acc):
```

Comment:
This is probably copied and pasted from elsewhere so I suggest moving it eva-common-pyutils. 
Also, you do not need the `assembly_name` here and it can be remove from the set.